### PR TITLE
{EventHub} `az eventhubs eventhub consumer-group create/update/delete/show/list`: Migrate eventhub consumer-group cmdlets to AAZ

### PR DESCRIPTION
**Pull request by schaudhari6254888**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
EventHub

**Comment by yonzhan** [comment]
Please resolve the conflicting files.

File: src/azure-cli/azure/cli/command_modules/eventhubs/aaz/latest/eventhubs/eventhub/consumer_group/__cmd_group.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "eventhubs eventhub consumer-group",
+)
+class __CMDGroup(AAZCommandGroup):
+    """eventhubs eventhub consumer-group
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/eventhubs/aaz/latest/eventhubs/eventhub/consumer_group/__cmd_group.py:18]: pls use meaningful description

Comment (schaudhari6254888) [reply src/azure-cli/azure/cli/command_modules/eventhubs/aaz/latest/eventhubs/eventhub/consumer_group/__cmd_group.py]: Done

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**